packages/bolt-connection/test/bolt/bolt-protocol-v5x0.test.js

```diff
@@ -587,6 +589,102 @@ describe('#unit BoltProtocolV5x0', () => {
 
       expect(unpackedDateTimeWithoutOffset).toEqual(object)
     })
+
+    it('should pack and unpack DateTimeWithOffset', () => {
+      fc.assert(
+        fc.property(
+          fc.date({
+            min: temporalUtil.newDate(utils.MIN_UTC_IN_MS + utils.ONE_DAY_IN_MS),
+            max: temporalUtil.newDate(utils.MAX_UTC_IN_MS - utils.ONE_DAY_IN_MS)
+          }),
+          utils.arbitraryTimeZoneId(),
+          (date, timeZoneId) => {
+            const object = new DateTime(
+              date.getUTCFullYear(),
+              date.getUTCMonth() + 1,
+              date.getUTCDate(),
+              date.getUTCHours(),
+              date.getUTCMinutes(),
+              date.getUTCSeconds(),
+              date.getUTCMilliseconds() * 1_000_000,
+              undefined,
+              timeZoneId
+            )
+            const buffer = alloc(256)
+            const loggerFunction = jest.fn()
+            const protocol = new BoltProtocolV5x0(
+              new utils.MessageRecordingConnection(),
+              buffer,
+              {
+                disableLosslessIntegers: true
+              },
+              undefined,
+              new Logger('debug', loggerFunction)
+            )
```

Comment:
Why is this not in a beforeEach hook for 5.0 like for the other protocol versions?

Reply:
We don't have a subset of test for this scenario in the v5